### PR TITLE
Update scorm ui

### PR DESCRIPTION
**Pull request by BenHSmith13**
Well, it updates the scorm ui

### CODE REVIEW

File: client/js/scorm/components/scorm/course.jsx [hunk 1]
```diff
@@ -37,45 +37,77 @@ export default class Course extends React.Component {
...
 
   handleGoClick(){
-    // TODO: need to have an action that sets the import type for the course.
-    if(this.state.isGoBtnActive){
-      const params = this.props.importPackage(
-        this.props.course.id,
-        this.props.course.title
-      );
-      this.setState({isGradeActive: true});
+    if(this.props.course.is_graded == Course.ImportTypes.GRADED){
```
Comment (BenHSmith13) [inline client/js/scorm/components/scorm/course.jsx]: let pointsPossible = 0;

File: client/js/scorm/components/scorm/course.jsx [hunk 2]
```diff
@@ -37,45 +37,77 @@ export default class Course extends React.Component {
...
+
+    this.props.importPackage(
+      this.props.course.id,
+      this.props.course.title,
+      this.props.course.index,
+      pointsPossible
+    );
+  }
+
+  formatGraded(course) {
```
Comment (BenHSmith13) [inline client/js/scorm/components/scorm/course.jsx]: delete me!

File: client/js/scorm/components/scorm/course.jsx [hunk 3]
```diff
@@ -37,45 +37,77 @@ export default class Course extends React.Component {
...
+    if(this.props.course.fetching){
+      dropDown = <div style={this.getStyles().loaderContainer}><Loader/></div>;
+    } else if(isAssignment && isGraded){
+      assignmentButton = <AssignmentButton {...assignmentButtonProps} />;
+      dropDown = <div className="c-list-item__type" style={{minWidth: "20rem"}}>Graded Assignment</div>;
+    } else if(isAssignment && !isGraded) {
+      assignmentButton = <AssignmentButton {...assignmentButtonProps} />;
+      dropDown = <div className="c-list-item__type" style={{minWidth: "20rem"}}>Ungraded Assignment</div>;
+    } else {
+      var isNotUnselected = this.props.course.is_graded == undefined || this.props.course.is_graded == Course.ImportTypes.NOT_SELECTED;
```
Comment (BenHSmith13) [inline client/js/scorm/components/scorm/course.jsx]: change to isSelected `this.props.course.is_graded != undefined && this.props.course.is_graded != Course.ImportTypes.NOT_SELECTED`

File: client/js/scorm/reducers/scorm.js [hunk 4]
```diff
@@ -8,10 +8,34 @@ const initialState = {
 export default (state = initialState, action) => {
   switch(action.type){
 
+    case "CREATE_ASSIGNMENT":
+      var updatedScorm = {...state.scormList[action.localOptions.index]};
+      updatedScorm.fetching = true;
+      var updatedScormList = state.scormList.slice();
```
Comment (BenHSmith13) [inline client/js/scorm/reducers/scorm.js]: use lets

**Comment by BenHSmith13** [comment]
LGTM

**Comment by jbasdf** [comment]
Looks good

**Pull request merged**